### DISCUSSION TITLE
Where is lam_f stored?

### DISCUSSION THREAD

**Discussion by brunomorampc** [casadi/casadi#3331] [Q&A] [answered]
I'd like to access the value of lam_f in the optistack debugger ( to use it then in the hessian of the lagrangian see my other discussion https://github.com/casadi/casadi/discussions/3330) . I can find lam_g but not lam_f?

**Comment by jgillis** [comment, marked as answer]
Not exposed, just choose lam_f = 1, that corresponds to the classic definition of Lagrangian
